linetools/spectralline.py

```diff
@@ -242,46 +256,31 @@ def fill_data(self,trans, linelist=None):
                        } )
 
     # Perform AODM on the line
-    def aodm(self, **kwargs): 
+    def aodm(self, flg_sat=None):
```

Comment:
Would it be better to pull this AODM code out of the spectral line class into its own function or module? Might be useful to be able to access it without have to create a SpectralLine.